readme.md

```diff
@@ -7,11 +7,17 @@
 
 ginh is not a histogram
 
-usage: `./ginh.sh [-h] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
+## Usage
 
-`ginh.sh` generates a bar chart of your most frequently used shell commands,
-according to your shell's history file (or another file of your choosing, using
-the `-f` flag). 
+usage: `ginh [-h] [-a] [-n entries] [-f hist_file] [-c chart_char] [-l line_len]`
+
+`ginh` generates a bar chart of your most frequently used shell commands,
+according to your shell's history file.
+
+other flags include `-n` for specifying the number of bars, `-c` to select
+the character used in each bar, `-l` to change the maximum line length
+(bar "height"), and `-a` to treat aliases as first class entries (default
+behavior is to revert aliases into their target command).
```

Comment:
the `-f` flag description is missing now

Reply:
I think I'm just going to replace this paragraph with the neater, more readable usage print out from the program.